util/cmdgen.py

```diff
@@ -74,6 +74,19 @@ def cmdgen_rewrite_md(filepath: Path, dry_run: bool, update: bool) -> bool:
         match_start_linum = 1 + content[0:match_start.start(0)].count("\n")
         cmd = match_start.group('cmd').strip()
 
+        # The command may contain paths, which are relative to the root of the
+        # repository. In order to make command invocation independent of the
+        # current working directory, we need to make these paths absolute.
+        cmd_parts = cmd.split(' ')
+        for idx, part in enumerate(cmd_parts):
+            # The first part (index 0) of a command is a path to an executable;
+            # all other parts are treated as paths if they contain a slash.
+            if idx == 0 or '/' in part:
```

Comment:
`'/i in part` is somewhat of a poor man's path detection, I think. If you guys have a better idea on reliably determining if a string is or could be a path, please suggest it.